testsuite/features/reposync/srv_add_rocky8_repositories.feature

```diff
@@ -49,13 +49,13 @@ Feature: Add the Rocky 8 distribution custom repositories
     And I select the "rocky-8-iso-baseos" repo
     And I click on "Save Repositories"
     Then I should see a "repository information was successfully updated" text
+    And I wait until the channel "rocky-8-iso" has been synced
```

Comment:
Putting the timeout here makes more sense and did work for me in my test environment.